package.json

```diff
@@ -58,46 +58,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^19.1.2",
-    "@angular/common": "^19.1.2",
-    "@angular/compiler": "^19.1.2",
-    "@angular/core": "^19.1.2",
-    "@angular/elements": "^19.1.2",
-    "@angular/forms": "^19.1.2",
-    "@angular/platform-browser": "^19.1.2",
-    "@angular/platform-browser-dynamic": "^19.1.2",
-    "@angular/platform-server": "^19.1.2",
-    "@angular/router": "^19.1.2",
-    "@angular/ssr": "^19.1.3",
+    "@angular/animations": "^19.1.6",
+    "@angular/common": "^19.1.6",
+    "@angular/compiler": "^19.1.6",
+    "@angular/core": "^19.1.6",
+    "@angular/elements": "^19.1.6",
+    "@angular/forms": "^19.1.6",
+    "@angular/platform-browser": "^19.1.6",
+    "@angular/platform-browser-dynamic": "^19.1.6",
+    "@angular/platform-server": "^19.1.6",
+    "@angular/router": "^19.1.6",
+    "@angular/ssr": "^19.1.7",
     "@igniteui/material-icons-extended": "^3.1.0",
     "@lit-labs/ssr-dom-shim": "^1.2.1",
     "@types/source-map": "0.5.2",
     "express": "^4.21.1",
     "fflate": "^0.8.1",
-    "igniteui-theming": "^14.3.1",
+    "igniteui-theming": "^14.5.0-beta.4",
```

Comment:
@simeonoff , @desig9stein , can we have a final 14.5 version to use here before merging?